android/sdl_android/src/main/java/com/smartdevicelink/managers/lifecycle/LifecycleManager.java

```diff
@@ -134,7 +134,7 @@ void onTransportDisconnected(String info, boolean availablePrimary, BaseTranspor
      * @param parameters  the desired video streaming parameters
      */
     @Override
-    void startVideoService(boolean isEncrypted, VideoStreamingParameters parameters) {
+    void startVideoService(boolean isEncrypted, VideoStreamingParameters parameters, boolean afterPendingRestart) {
```

Comment:
@kostyaBoss 
Please update the documentation above to include the afterPendingRestart parameter
`@param afterPendingRestart specify true when the videoService is starting again after the display has been resized`